src/others/components/map/CircleLayerStyleWithRegions.tsx

```diff
@@ -0,0 +1,41 @@
+import { LayerProps } from "react-map-gl";
+
+export const layerStyle: LayerProps = {
+  id: "ukr_water_needs-point",
+  type: "circle",
+  minzoom: 2,
+  paint: {
+    "circle-radius": [
+      "interpolate",
+      ["linear"],
+      ["zoom"],
+      7.8,
+      ["interpolate", ["linear"], ["get", "normalized_amount"], 0, 0, 1, 0],
+      7.9,
+      ["interpolate", ["linear"], ["get", "normalized_amount"], 0, 0, 1, 10],
+      16,
+      ["interpolate", ["linear"], ["get", "normalized_amount"], 0, 0, 1, 200],
+    ],
+
+    "circle-color": [
+      "interpolate",
+      ["linear"],
+      ["get", "normalized_amount"],
+      0,
+      "rgba(33,102,172,0)",
```

Comment:
I opt for writing even rough explanations in a comment (maybe one, above the whole const). It would be good for people that will jump into this code. We could also add a link to the official documentation there. 

About creating variables I'm not sure if it's worth it. I also don't see any pros of having a possibility to manipulate those numbers from a parent component perspective. For me, they are just hardcoded values that will stay unchanged during the lifetime of an application.

Reply:
Now we'll have at least 3 files where this interpolation will be used, do you suggest to having the comment in all three places?

Reply:
What if it grows to 5-10 places. What if one decides to update that comment, do you need to go an update all 10 places?
I don't believe a comment in this case is needed, since it takes a simple google search "maplibre interpolate" to get to the official documentation.
And if we start adding comments for every call to an external API that someone is not familiar with, we'll have 99% of the code being comments.

Looking forward to hearing your arguments :)

Reply:
So maybe a simple oneliner with a link to the doc? It won't have to be updated and we could simply copy-paste it. If someone will forget to add this comment with the creation of a new style it won't hurt us so much but still, there is a higher chance that someone will copy it from an existing file. 

It would be nice to know what to type in the google search bar :)